docs/ruleset_functions/get_spaces_served_by_SWH_use.md

```diff
@@ -0,0 +1,39 @@
+## get_spaces_served_by_SWH_use
+
+Description: This function determines the spaces served by a given SWH use.  The rule is that if spaces reference the swh_use, then the use applies to only those spaces.  If the use references no spaces, it applies to all spaces in the building segment.
+
+
+Inputs:
+- **RMD**
+- **swh_use**
+
+Returns:
+- **spaces_served**: a list of space ids
+
+Function Call:
+
+
+Data Lookup: None
+
+Logic:
+
+- find the building_segment that contains this swh_use by looking through each building segment: `for building_segment in RMD.building_segments:`
+    - check if the swh_use is in this building segment: `if swh_use in building_segment.service_water_heating_uses:`
+        - create a list of spaces: `spaces = []`
+        - look at each building segment: `for building_segment in RMD.building_segments`
+            - check if the swh_use is in this building_segment: `if swh_use in building_segment.service_water_heating_uses:`
+                - look at each space in the building segment to see which of them reference the swh_use: `for space in building_segment.spaces:`
```

Comment:
`for space in building_segment...spaces:` add extra dots for clarity to indicate that spaces are not a direct child of building segment